linux/keyman-config/keyman_config/downloadkeyboard.py

```diff
@@ -30,20 +30,42 @@ def __init__(self, parent=None):
         s = Gtk.ScrolledWindow()
         self.webview = WebKit2.WebView()
         self.webview.connect("decide-policy", self._keyman_policy)
+        self.webview.connect("load-changed", self._update_back_button)
         url = KeymanComUrl + "/go/linux/" + __releaseversion__ + "/download-keyboards"
         self.webview.load_uri(url)
         s.add(self.webview)
 
         self.get_content_area().pack_start(s, True, True, 0)
 
-        self.add_button(_("_Close"), Gtk.ResponseType.CLOSE)
+        hbox = Gtk.Box(spacing=6)
+        self.get_content_area().pack_start(hbox, False, False, 0)
+
+        self.back_button = Gtk.Button.new_with_mnemonic(_("_Back"))
+        self.back_button.set_tooltip_text(_("Back to search"))
+        self.back_button.connect("clicked", self._on_back_clicked)
+        self.back_button.set_sensitive(False)
+        hbox.pack_start(self.back_button, False, False, 0)
+
+        close_button = Gtk.Button.new_with_mnemonic(_("_Close"))
```

Comment:
Do these strings get localized? (tooltip and button labels)

Reply:
Yes